### PR TITLE
Separate APIs

### PR DESCRIPTION
**Pull request by admiral96**

### CODE REVIEW

File: app/geocode_api_parser.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        self.formatted_address = None
+        self.place_type = None
+
+    def get_address_query(self):
+        params = {'address': self.address, 'key': GEOCODE_API_KEY}
+        return json.loads(requests.get('{}'.format(geocode_url), params=params).text)
+
+    def is_place_query_valid(self):
+        status = self.get_address_query()['status']
+        return True if status == 'OK' else False
```
Comment (mknecht) [inline app/geocode_api_parser.py]: why not just `return status == 'OK'` ?

File: app/geocode_api_parser.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+class Geocode:
+
+    def __init__(self, address):
+        self.address = address
+        self.formatted_address = None
+        self.place_type = None
+
+    def get_address_query(self):
+        params = {'address': self.address, 'key': GEOCODE_API_KEY}
+        return json.loads(requests.get('{}'.format(geocode_url), params=params).text)
```
Comment (mknecht) [inline app/geocode_api_parser.py]: `'{}'.format(geocode_url) == geocode_url`

:)

File: app/geocode_api_parser.py (hunk 1 above)
Comment (admiral96) [inline app/geocode_api_parser.py]: I wanted the function to only return boolean, reflects its name

Comment (admiral96) [inline app/geocode_api_parser.py]: Okay, got it! Sorry, I thought the other way around :P

**Pull request merged**